### DISCUSSION TITLE
SCSS File Font Guidance

### DISCUSSION THREAD

**Discussion by adamebell** [Venipa/ytmdesktop2#142] [General]
Hi, thanks for making this app. I was wondering if you had any guidance for changing the font with the SCSS file? I've tried a few basic things but didn't have any success. With the update, my player has a serif font that I don't love but figured the new SCSS feature would be the best way for me to change it.

Thanks!

**Comment by Venipa** [comment]
currently importing fonts from url (for example google font api or other http urls) is currently broken due to electron changes and csp
ill see what i can do

**Reply by adamebell** [reply]
Ah that makes sense. No rush, it's just cosmetic & the app works perfectly otherwise.
